<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glwidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>640</width>
      <height>505</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Open Tecplot file</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set number of interpolation (bilinear)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Draw color image</string>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Data component for colors and isolines</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Draw isolines</string>
    </property>
    <property name="text">
     <string>Isolines</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>0</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of isolines (linear)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Draw vectors</string>
    </property>
    <property name="text">
     <string>Vectors</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of skipped values</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Data component for first vector axis</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>0</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Data component for second vector axis</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>glwidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
